app/src/main/java/com/spendster/presentation/mainScreen/HomeActivity.java

```diff
@@ -0,0 +1,92 @@
+package com.spendster.presentation.mainScreen;
+
+import android.content.res.ColorStateList;
+import android.graphics.Color;
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.support.constraint.ConstraintLayout;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewCompat;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.spendster.R;
+import com.spendster.presentation.welcomeScreen.SlideFragment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HomeActivity extends AppCompatActivity implements View.OnClickListener, BottomNavigationView.OnNavigationItemSelectedListener {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        setTheme(R.style.AppTheme);
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home);
+        initUI();
+    }
+
+    private void initUI() {
+        final Toolbar toolbar = findViewById(R.id.homeToolbar);
+        setSupportActionBar(toolbar);
+        final BottomNavigationView bottomNavigationView = findViewById(R.id.bottomNavigationMenu);
+        bottomNavigationView.setOnNavigationItemSelectedListener(this);
+        loadFragment(new ActivityFragment());
+        initButtons();
+    }
+
+    private void initButtons(){
+        Button btnAdd = findViewById(R.id.btnAdd);
+        btnAdd.setOnClickListener(this);
+    }
+
+    @Override
+    public void onClick(View view) {
+        switch (view.getId()){
+            case R.id.btnAdd:
+                Toast.makeText(this, "Add Expenses", Toast.LENGTH_SHORT).show();
+        }
+
+    }
+
+    @Override
+    public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
+        Fragment fragment = null;
+        switch (menuItem.getItemId()){
+            case R.id.activityItem:
+                fragment = new ActivityFragment();
```

Comment:
@SimonMaha It's better to use this method to instantiate fragments:
https://www.androiddesignpatterns.com/2012/05/using-newinstance-to-instantiate.html